spotbugs/etc/messages.xml

```diff
@@ -124,7 +124,7 @@ This plugin contains all of the standard SpotBugs detectors.
   <BugCategory category="STYLE">
     <Description>Dodgy code</Description>
     <Abbreviation>D</Abbreviation>
-    <Details>Code that is confusing, anomalous, or
```

Comment:
This seems better capitalised.

Reply:
I left the merging to my IDE which caused a bit of chaos. I looked at it manually, hopefully it's all good now!